<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>havadurumu</class>
 <widget class="QMainWindow" name="havadurumu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>havadurumu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(183, 233, 255);


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="cityLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 249, 217);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="CelciusButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
    color: black; 
    font-weight: bold;
}

QRadioButton::indicator {
    width: 14px;
    height: 14px;
    border-radius: 7px;               
    border: 2px solid #222;           
    color: black;
    background-color: rgb(183, 233, 255);
}

QRadioButton::indicator:checked {
    background-color:#888888;          
    border: 2px solid #000;          
}
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FahreneitButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {
    color: black; 
    font-weight: bold;
}

QRadioButton::indicator {
    width: 14px;
    height: 14px;
    border-radius: 7px;               
    border: 2px solid #222;           
    color: black;
    background-color: rgb(183, 233, 255);
}

QRadioButton::indicator:checked {
    background-color:#888888;          
    border: 2px solid #000;          
}</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fetchButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 249, 217);
 font-weight: bold;</string>
    </property>
    <property name="text">
     <string>RECEIVE</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="resultScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>511</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>509</width>
       <height>519</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="resultLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: black;
background-color: rgb(255, 249, 217);


</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 39, 39);
 font-weight: bold;</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherImageLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 249, 217);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="weatherInfoLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 249, 217);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="suggestionButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>430</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 249, 217);</string>
    </property>
    <property name="text">
     <string>SUGGEST</string>
    </property>
   </widget>
   <widget class="QLabel" name="suggestionLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 249, 217);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="suggestionLabel_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 249, 217);</string>
    </property>
    <property name="text">
     <string>Please press for event suggestion =&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
